docs/data_collection.rst

```diff
@@ -0,0 +1,99 @@
+Data Collection
+===============
+
+This document describes how snippets data is collected, stored and used.
+
+Snippets follows Mozilla's established best practices for data collection
+and storage of personally identifiable information.
+
+Data Collection: Impressions
```

Comment:
No need to prefix with "Data Collection" when it's already the title of the doc.